patches/server/0881-Prevent-entity-loading-causing-async-lookups.patch

```diff
@@ -0,0 +1,68 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Sun, 6 Mar 2022 11:09:09 -0500
+Subject: [PATCH] Prevent entity loading causing async lookups
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/Entity.java b/src/main/java/net/minecraft/world/entity/Entity.java
+index 84fce7dccf9232209f939a32acfc3131e62eb27c..34847add2a35e535b001f581677c4953dd384800 100644
+--- a/src/main/java/net/minecraft/world/entity/Entity.java
++++ b/src/main/java/net/minecraft/world/entity/Entity.java
+@@ -725,6 +725,7 @@ public abstract class Entity implements Nameable, EntityAccess, CommandSource, i
+ 
+     public void baseTick() {
+         this.level.getProfiler().push("entityBaseTick");
++        if (firstTick && this instanceof net.minecraft.world.entity.NeutralMob neutralMob) neutralMob.tickInitialPersistentAnger(level); // Paper - Update last hurt when ticking
+         this.feetBlockState = null;
+         if (this.isPassenger() && this.getVehicle().isRemoved()) {
+             this.stopRiding();
+diff --git a/src/main/java/net/minecraft/world/entity/NeutralMob.java b/src/main/java/net/minecraft/world/entity/NeutralMob.java
+index dedf76de5d6f46b9626ca4a98cfffe125b90dd0c..f02224c1993b7953f6d7c4bad1edc6c41cc8bbf2 100644
+--- a/src/main/java/net/minecraft/world/entity/NeutralMob.java
++++ b/src/main/java/net/minecraft/world/entity/NeutralMob.java
+@@ -42,18 +42,6 @@ public interface NeutralMob {
+                 UUID uuid = nbt.getUUID("AngryAt");
+ 
+                 this.setPersistentAngerTarget(uuid);
+-                Entity entity = ((ServerLevel) world).getEntity(uuid);
+-
+-                if (entity != null) {
+-                    if (entity instanceof Mob) {
+-                        this.setLastHurtByMob((Mob) entity);
+-                    }
+-
+-                    if (entity.getType() == EntityType.PLAYER) {
+-                        this.setLastHurtByPlayer((Player) entity);
+-                    }
+-
+-                }
```

Comment:
Add a replacement paper comment just saying that this was moved to its own method to do earlier.

Reply:
Lmk if I did this right.